packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -31,6 +32,8 @@ export class DataSyncPlugin extends GraphbackPlugin {
 
       return schema;
     }
+
+    this.addMetaFields(schemaComposer, models);
```

Comment:
With this call, we are introducing a double loop. 

Do you think we can perform the actions in https://github.com/aerogear/graphback/pull/1513/files#diff-37ea145aded7cc3fa4e8f167fac3af2bR140 in the this forEach loop? https://github.com/aerogear/graphback/pull/1513/files#diff-37ea145aded7cc3fa4e8f167fac3af2bR37 

What I have in mind is to the `addMetaFields` (which is to be named) method could instead take the model and not the list of models.

Reply:
I see, Thanks! :D